test/functional/testUtil.go

```diff
@@ -73,7 +73,7 @@ func SetDefault() (string, string) {
 	defaultDockerReg := os.Getenv("DOCKER_REGISTRY")
 	imageTag := os.Getenv("REL_VERSION")
 	if defaultDockerReg == "" {
-		defaultDockerReg = "ghcr.io/radius-project/dev"
+		defaultDockerReg = "ghcr.io/radius-project"
```

Comment:
this should still work because we set the `DOCKER_REGISTRY` environment variable to `ghcr.io/radius-project/dev` on PR. I think the only time the default is actually being used is on local functional tests runs.